src/pages/Rockets.jsx

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable no-unused-vars */
+import React, { useState, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  getRocketsArr, reserveRocket, unreserveRocket,
+} from '../redux/rockets/rocketsSlice';
+
+function RocketsFunction() {
+  const dispatch = useDispatch();
+  const rocketsArr = useSelector((state) => state.rockets.rocketsArr);
+  const ifSucceed = useSelector((store) => store.rockets.ifSucceed);
+  const [type, setType] = useState('');
+  const [name, setName] = useState('');
```

Comment:
- [ ] instead of disabling n0-unsed var linters you should remove this lines of code as it was not used :+1: